### PR TITLE
docs: initial check-in for ngrx.io project

### PR DESCRIPTION
**Pull request by brandonroberts**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

## Other information

Initial setup of docs app for NgRx.io. Content still a WIP but this gets us on the way.

### CODE REVIEW

**Comment by timdeschryver** [comment]
How "far" do you want this review to be?
Because I see that there are references to angular.io and that there are some doc pages without content, but I don't know if these are in scope of this PR.

**Comment by brandonroberts** [comment]
Those are outside the scope of this PR as they will get cleaned up if required. The angular.io references in the folders is because its using the same docs generation tooling. This is just to get the initial page up so we can start migrating/organizing the content.

**Comment by timdeschryver** [comment]
Ok that's what I thought just wanted to make sure 😄

**Comment by brandonroberts** [comment]
Ok. We won't need the symlink stuff since we don't use any Angular source here. I'll clean up the README to remove all the stuff that's not relevant.

**Comment by brandonroberts** [comment]
The scrollbar and README should be fixed now.

File: projects/ngrx.io/content/marketing/contributors.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    "group": "Core"
+  },
+
+  "brandon": {
+    "name": "Brandon Roberts",
+    "picture": "brandonroberts.jpg",
+    "twitter": "brandontroberts",
+    "website": "http://github.com/brandonroberts",
+    "bio":
+      "Brandon is member of the Angular Team working on documentation and development. He",
```
Comment (timdeschryver) [inline projects/ngrx.io/content/marketing/contributors.json]: He ...

**Comment by timdeschryver** [comment]
The next version button goes to the angular docs.
image

**Pull request merged**